app/services/idv/steps/verify_wait_step_show.rb

```diff
@@ -28,18 +28,21 @@ def process_async_state(current_async_state)
 
       def async_state_done(current_async_state)
         add_proofing_costs(current_async_state.result)
-        response = idv_result_to_form_response(current_async_state.result)
-        response = check_ssn(flow_session[:pii_from_doc]) if response.success?
-        summarize_result_and_throttle_failures(response)
+        form_response = idv_result_to_form_response(current_async_state.result)
+        if form_response.success?
```

Comment:
the changes here are the key, since we were previously discarding all of the response data from the proofing result by assigning `response` to the result of `check_ssn`.

The merge preserves errors/success from `check_ssn` while making sure the context from the proofing result gets logged.